.gitignore

```diff
@@ -13,3 +13,4 @@ npm-debug.log
 **/.settings/
 **/.project
 **/.classpath
+blueocean-web/src/main/webapp/
```

Comment:
We just want the folder there for holding the assets, which get put into it at build time. We don't want any of the contents checked in.